.cirrus.yml

```
@@ -25,10 +25,13 @@ final_docker_builder:
   only_if: $CIRRUS_BRANCH == "main"
   env:
     GITHUB_TOKEN: ENCRYPTED[98160948dd9e4613e0f7704958694e53fd7f021dd847938991e91c80e14ac5c91822aa9dff3fa34e52ec76f331beeeb5]
+  get_date_script: echo "DATE_TAG=$(date +%d%m%Y-%H%M%S)" >> $CIRRUS_ENV
```

It takes about 20 minutes to build the dev container, so I don't think we need a seconds tag.  Hour and minute is probably fine.  

I'd prefer that we used ISO date order so that tags sorted lexically are in chronological order.

```suggestion
    get_date_script: echo "DATE_TAG=$(date +%Y-%m-%d-%H.%M)" >> $CIRRUS_ENV
```

I think this is also indented incorrectly (YAML is whitespace sensitive).

Note that this will, I believe, be evaluated when the script is run, so will be different for the different runs.

---

Why should it be indented, its not part of the "env:" block but a separate script to create an additional env var in $CIRRUS_ENV

If I'm reading the cirrus docs right that seems to be the suggested way to make an env value available across different tasks:  https://cirrus-ci.org/guide/tips-and-tricks/